tools/src/wyvern/stdlib/support/CommandLineUtils.java

```diff
@@ -0,0 +1,30 @@
+package wyvern.stdlib.support;
+
+public class CommandLineUtils {
+    private static String[] arguments; // array of command line arguments.
+    private static DynArrayList argumentsArrayList; // array list of command line arguments.
+    public static final CommandLineUtils utils = new CommandLineUtils();
+
+    private CommandLineUtils() {
+    }
+
+    public CommandLineUtils(String[] arguments) {
```

Comment:
This design--where there are two constructors, and this one is used only for its side effect on global variables--is hard to understand.  It would be better to omit this constructor and instead just call convertToDynArrayList() from the interpreter's main function, passing the arguments.